FE/src/components/domain/AgentCard.tsx

```
@@ -1,70 +1,122 @@
   className: string;
 }) => {
   const context = useContext(AuthContext);
+  const { resizeImage, imageSize, loadingImage } = useResizeImage();
   if (!context) return null;
   const { userType } = context;
 
   const animationStyle = hasAnimation
     ? "transition-all duration-150 ease-out hover:scale-107 hover:m-2"
     : "";
+
+  useEffect(() => {
```

Biome (2.3.14)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)